makefile

```diff
@@ -91,29 +91,29 @@ start_sentinel: ${BINARY}
 	done
 
 stop_cluster:
-	$(call kill-redis,${CLUSTER_PID_PATHS})
-	rm -f appendonly.aof || true
-	rm -f ${CLUSTER_CONF_PATHS} || true
+	@$(call kill-redis,${CLUSTER_PID_PATHS})
+	@rm -f appendonly.aof || true
+	@rm -f ${CLUSTER_CONF_PATHS} || true
 
 start_cluster: ${BINARY}
-	for port in ${CLUSTER_PORTS}; do                    \
-		${BINARY}                                         \
-			--daemonize            yes                      \
-			--appendonly           yes                      \
-			--cluster-enabled      yes                      \
-			--cluster-config-file  ${TMP}/nodes$$port.conf  \
-			--cluster-node-timeout 5000                     \
-			--pidfile              ${TMP}/redis$$port.pid   \
-			--port                 $$port                   \
-			--unixsocket           ${TMP}/redis$$port.sock; \
+	@for port in ${CLUSTER_PORTS}; do\
+		${BINARY}\
+			--daemonize            yes\
+			--appendonly           yes\
+			--cluster-enabled      yes\
+			--cluster-config-file  ${TMP}/nodes$$port.conf\
+			--cluster-node-timeout 5000\
+			--pidfile              ${TMP}/redis$$port.pid\
+			--port                 $$port\
+			--unixsocket           ${TMP}/redis$$port.sock;\
 	done
+	@sleep 3
```

Comment:
```
make create_cluster
make[2]: Entering directory `/home/travis/build/supercaracal/redis-rb'
yes yes | ((tmp/cache/redis-5.0/src/redis-cli --cluster create 127.0.0.1:7000 127.0.0.1:7001 127.0.0.1:7002 127.0.0.1:7003 127.0.0.1:7004 127.0.0.1:7005 --cluster-replicas 1) || \
		(bundle exec ruby tmp/cache/redis-5.0/src/redis-trib.rb create --replicas 1 127.0.0.1:7000 127.0.0.1:7001 127.0.0.1:7002 127.0.0.1:7003 127.0.0.1:7004 127.0.0.1:7005))
Could not connect to Redis at 127.0.0.1:7005: Connection refused
```

Reply:
Is there any other way than sleep? e.g. poll with  a `redis-cli ping`

Reply:
Certainly, we might be able to use [reconnection](https://github.com/redis/redis-rb#reconnections) feature. I'll fix it.

Reply:
done